2018-05-25 15:14:23 [main] TRACE ExecuteQuery:65 - ~~~~~~~~~~~~~~~STARTING-2018.05.25 15.14.23~~~~~~~~~~~~~~~
2018-05-25 15:14:24 [main] TRACE ConnectToServer:59 - ====================START-PREPROD====================
2018-05-25 15:14:25 [main] INFO  ConnectToServer:39 - ssh ashnavsaxena@bastion.lenskartserver.net:22
2018-05-25 15:14:25 [main] INFO  ConnectToServer:79 - Local port forwarding as 3311:mysql-scm-rw.preprod.internal:3306
2018-05-25 15:14:25 [main] INFO  ConnectToServer:69 - ssh established? true
2018-05-25 15:14:26 [main] INFO  ConnectToMysql:51 - JDBC Connection to jdbc:mysql://127.0.0.1:3311 established? true
2018-05-25 15:14:26 [main] ERROR ConnectToMysql:98 - Invalid file name or location. 
java.io.FileNotFoundException: src/main/resources/LocalFitting_Query.sql (No such file or directory)
	at java.io.FileInputStream.open0(Native Method) ~[?:1.8.0_121]
	at java.io.FileInputStream.open(FileInputStream.java:195) ~[?:1.8.0_121]
	at java.io.FileInputStream.<init>(FileInputStream.java:138) ~[?:1.8.0_121]
	at java.util.Scanner.<init>(Scanner.java:611) ~[?:1.8.0_121]
	at utilities.ConnectToMysql.executeSqlScript(ConnectToMysql.java:94) [classes/:?]
	at com.ashnav.pipeline.orderstate.ExecuteQuery.executeOnAllEvironments(ExecuteQuery.java:49) [test-classes/:?]
	at com.ashnav.pipeline.orderstate.ExecuteQuery.main(ExecuteQuery.java:66) [test-classes/:?]
2018-05-25 15:14:26 [main] INFO  ConnectToMysql:78 - JDBC Connection to jdbc:mysql://127.0.0.1:3311 destroyed? true
2018-05-25 15:14:26 [main] INFO  ConnectToServer:100 - 3311 local port forwarding destroyed
2018-05-25 15:14:26 [main] INFO  ConnectToServer:92 - ashnavsaxena@bastion.lenskartserver.net:22 logged out
2018-05-25 15:14:26 [main] TRACE ConnectToServer:62 - ====================END-PREPROD====================
2018-05-25 15:14:26 [main] TRACE ConnectToServer:59 - ====================START-SCMQE====================
2018-05-25 15:14:27 [main] INFO  ConnectToServer:39 - ssh ashnavsaxena@54.255.251.151:22
2018-05-25 15:14:27 [main] INFO  ConnectToServer:79 - Local port forwarding as 3312:mysql-scm-rw.scmqe.internal:3306
2018-05-25 15:14:27 [main] INFO  ConnectToServer:69 - ssh established? true
2018-05-25 15:14:28 [main] INFO  ConnectToMysql:51 - JDBC Connection to jdbc:mysql://127.0.0.1:3312 established? true
2018-05-25 15:14:28 [main] ERROR ConnectToMysql:98 - Invalid file name or location. 
java.io.FileNotFoundException: src/main/resources/LocalFitting_Query.sql (No such file or directory)
	at java.io.FileInputStream.open0(Native Method) ~[?:1.8.0_121]
	at java.io.FileInputStream.open(FileInputStream.java:195) ~[?:1.8.0_121]
	at java.io.FileInputStream.<init>(FileInputStream.java:138) ~[?:1.8.0_121]
	at java.util.Scanner.<init>(Scanner.java:611) ~[?:1.8.0_121]
	at utilities.ConnectToMysql.executeSqlScript(ConnectToMysql.java:94) [classes/:?]
	at com.ashnav.pipeline.orderstate.ExecuteQuery.executeOnAllEvironments(ExecuteQuery.java:49) [test-classes/:?]
	at com.ashnav.pipeline.orderstate.ExecuteQuery.main(ExecuteQuery.java:66) [test-classes/:?]
2018-05-25 15:14:28 [main] INFO  ConnectToMysql:78 - JDBC Connection to jdbc:mysql://127.0.0.1:3312 destroyed? true
2018-05-25 15:14:28 [main] INFO  ConnectToServer:100 - 3312 local port forwarding destroyed
2018-05-25 15:14:28 [main] INFO  ConnectToServer:92 - ashnavsaxena@54.255.251.151:22 logged out
2018-05-25 15:14:28 [main] TRACE ConnectToServer:62 - ====================END-SCMQE====================
2018-05-25 15:14:28 [main] TRACE ConnectToServer:59 - ====================START-WEBQE====================
2018-05-25 15:14:28 [main] INFO  ConnectToServer:39 - ssh ashnavsaxena@54.255.251.204:22
2018-05-25 15:14:28 [main] INFO  ConnectToServer:79 - Local port forwarding as 3313:mysql-scm-rw.webqe.internal:3306
2018-05-25 15:14:28 [main] INFO  ConnectToServer:69 - ssh established? true
2018-05-25 15:14:29 [main] INFO  ConnectToMysql:51 - JDBC Connection to jdbc:mysql://127.0.0.1:3313 established? true
2018-05-25 15:14:29 [main] ERROR ConnectToMysql:98 - Invalid file name or location. 
java.io.FileNotFoundException: src/main/resources/LocalFitting_Query.sql (No such file or directory)
	at java.io.FileInputStream.open0(Native Method) ~[?:1.8.0_121]
	at java.io.FileInputStream.open(FileInputStream.java:195) ~[?:1.8.0_121]
	at java.io.FileInputStream.<init>(FileInputStream.java:138) ~[?:1.8.0_121]
	at java.util.Scanner.<init>(Scanner.java:611) ~[?:1.8.0_121]
	at utilities.ConnectToMysql.executeSqlScript(ConnectToMysql.java:94) [classes/:?]
	at com.ashnav.pipeline.orderstate.ExecuteQuery.executeOnAllEvironments(ExecuteQuery.java:49) [test-classes/:?]
	at com.ashnav.pipeline.orderstate.ExecuteQuery.main(ExecuteQuery.java:66) [test-classes/:?]
2018-05-25 15:14:29 [main] INFO  ConnectToMysql:78 - JDBC Connection to jdbc:mysql://127.0.0.1:3313 destroyed? true
2018-05-25 15:14:29 [main] INFO  ConnectToServer:100 - 3313 local port forwarding destroyed
2018-05-25 15:14:29 [main] INFO  ConnectToServer:92 - ashnavsaxena@54.255.251.204:22 logged out
2018-05-25 15:14:29 [main] TRACE ConnectToServer:62 - ====================END-WEBQE====================
2018-05-25 15:14:29 [main] TRACE ConnectToServer:59 - ====================START-POSQE====================
